### DISCUSSION TITLE
Node resizing supported?

### DISCUSSION THREAD

**Discussion by dror-weiss** [xyflow/xyflow#3740] [Svelte]
The Node [docs](https://svelteflow.dev/api-reference/types/node#resizing) mention it is but I can't tell if it's a bug or not supported yet ...

**Comment by moklick** [comment]
The resizing flags indicates if a node is getting resized. If you want to resize nodes, you can use the `NodeResizer` component: https://reactflow.dev/examples/nodes/node-resizer

**Reply by dror-weiss** [reply]
So this is supported in React, while I was asking about Svelte.
You answered it's a flag, so thanks for that.

**Reply by moklick** [reply]
Got it! We will add it to Svelte Flow soon!

**Reply by moklick** [reply]
it's part of Svelte Flow since 0.0.32 - https://svelteflow.dev/api-reference/components/node-resizer
